### PR TITLE
Some more deprecation fixes

### PR DESCRIPTION
**Pull request by fredrikekre**

### CODE REVIEW

File: src/Documenter.jl [hunk 1]
```diff
@@ -34,6 +34,10 @@ else
     using Base: ip_matches_func
 end
 
+@static if VERSION >= v"0.7.0-DEV.3656"
+    import Pkg
+end
```
Comment (mortenpi) [inline src/Documenter.jl]: Wouldn't `import Compat: Pkg` work?

**Comment by fredrikekre** [comment]
Unrelated nightly windows failure, requires admin to merge.

**Comment by mortenpi** [comment]
Thanks! You would also want a tag soon-ish I guess?

**Comment by fredrikekre** [comment]
> You would also want a tag soon-ish I guess?

No hurry.

**Pull request merged**